config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -16,7 +16,6 @@ spec:
     listKind: KubeadmConfigList
     plural: kubeadmconfigs
     singular: kubeadmconfig
-  preserveUnknownFields: false
```

Comment:
We likely don't want to drop this, it may require updating the kubebuilder annotation for KubeadmConfig to set `crd:preserveUnknownFields=false` as per https://github.com/kubernetes-sigs/controller-tools/pull/362